ui/ts/pages/stores.ts

```diff
@@ -138,7 +141,24 @@ module AdminViews {
      * StorePage show the details of a single node.
      */
     export module StorePage {
+      import NavigationBar = Components.NavigationBar;
+
       class Controller {
+        private static defaultTargets: NavigationBar.Target[] = [
+          {
+            title: "Overview",
+            route: ""
+          },
+          {
+            title: "Graphs",
+            route: "graph"
+          }
+        ];
+
+        private static isActive: (targ: NavigationBar.Target) => boolean = (t: NavigationBar.Target) => {
```

Comment:
The route is not one of our properties, it's a mithril property - we can't depend on it with our computed properties.

Also, 'isActive' doesn't return the active target - it accepts a target, and returns true if it is active.  It's only used to add a css class to the active link.